src/scss/_about.scss

```diff
@@ -1,6 +1,5 @@
 //---------------Break Points---------------//
-//50em (800px)
-//61.25em (980px)
+$bp-about-small: 50em; //800px
```

Comment:
Looks like only your small breakpoint is defined here? All of your breakpoint variables should be defined in the same place.